ext/eft.py

```diff
@@ -0,0 +1,55 @@
+'''
+Cog that copies EFT-formatted fits into an embed.
+'''
+
+import logging
+import urllib.request, json, re
+
+from datetime import datetime
+
+import discord.ext.commands as commands
+import utils.checks as checks
+from discord.embeds import Embed
+
+def setup(bot):
+    'Adds the cog to the provided Discord bot'
+    bot.add_cog(EFT(bot))
+    
+class EFT:
+    'A cog that copies EFT-formatted fits to an embed'
+    def __init__(self, bot):
+        self.logger = logging.getLogger(__name__)
+        self.bot = bot
+    
+    async def on_message(self, message):
+        content = str(message.content)
+        p = re.compile('\[(.*?)\]\n(.*)', re.DOTALL)

```

Comment:
This regex is far too general. This will basically end up deleting any message containing square brackets (assuming the eftembed method doesn't raise an exception)